lib/responses/qtb.py

```diff
@@ -0,0 +1,206 @@
+import asyncio
+import discord
+import datetime
+import random
+import os
+import time
+
+from sqlalchemy.orm.exc import NoResultFound
+from lib.client import client
+from lib.models import QtAnimeGirl, Tag, session
+from hakase import config
+
+# Channels where qtb is in progress
+ongoing_qtb = []
+
+async def get_new_girls(message, params):
+  anime_girl = QtAnimeGirl()
+  new_girls = anime_girl.get_new_girls(config['image_directory'])
+  await client.send_message(message.channel, "{} new girls added!".format(new_girls))
+
+def _check_for_qtb_command(message):
+  command = message.content.split(' ')[0]
+
+  if command in ['>vote', '>end', '>name', '>tag', '>tags']:
+    return True
+  else:
+    return False
+
+async def _name_girl(message, girls):
+  index = int(message.content[[char.isdigit() for char in message.content].index(True)]) - 1
+  girl = girls[index]
+
+  name = message.content[5:]
+
+  girl.name = name
```

Comment:
consider adding limitations to this, such as max length or preventing people from giving them @ discord names